.github/ISSUE_TEMPLATE/improvements.md

```diff
@@ -13,10 +13,10 @@ Describe more about the change.
 ### Domain of Docs
 
```

Comment:
after line 13 add this 
And in order to tick the check box just but x inside them for example - [x] like this. Please delete options that are not relevant.
as a comment